keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -1482,10 +1539,34 @@ type CELDeviceSelector struct {
     //
     //     cel.bind(dra, device.attributes["dra.example.com"], dra.someBool && dra.anotherBool)
     //
+    // The length of the expression must be smaller or equal to 10 Ki. The
+    // cost of evaluating it is also limited based on the estimated number
+    // of logical steps. Validation against those limits happens only when
+    // setting an expression for the first time or when changing
+    // it. Therefore it is possible to change these limits without
+    // affecting stored expressions. Those remain valid.
```

Comment:
@jpbetz, @cici37: can you help review this proposal for a CEL cost limit in DRA? See also below for more comments on `CELSelectorExpressionMaxCost`.

The corresponding code is currently in https://github.com/kubernetes/kubernetes/pull/127511.

Reply:
IMHO this is not a blocker for merging the KEP. We can still review the details in the code PR.

Reply:
agreed